verify_ux.py

```
@@ -0,0 +1,30 @@
+    row = page.locator(".row").filter(has=heading).first
+
+    # Take screenshot of the row
+    print("Taking screenshot...")
+    row.screenshot(path="verification.png")
+
+with sync_playwright() as p:
+    browser = p.chromium.launch()
+    page = browser.new_page()
+    try:
+        verify_homepage(page)
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)